src/a-d/Alligator/Alligator.Catalog.cs

```diff
@@ -0,0 +1,40 @@
+namespace Skender.Stock.Indicators;
+
+public static partial class Alligator
+{
+    // Alligator Series Listing
+    public static readonly IndicatorListing SeriesListing =
+        new IndicatorListingBuilder()
+            .WithName("Williams Alligator")
+            .WithId("ALLIGATOR")
+            .WithStyle(Style.Series)
+            .WithCategory(Category.Oscillator)
```

Comment:
Ensure you’re picking the right category for all of these indicators, you can uses these as resources for info:
1. Our documentation: /docs/_indicators
2. An old incomplete JSON catalog file: /tests/indicators/_common/Catalog/catalog.bak.json